### PR TITLE
upgrade: add ignoring resource for modified flag from fleet-agent

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
related with https://github.com/harvester/harvester-installer/pull/345

original issue: https://github.com/harvester/harvester/issues/2762

### CODE REVIEW

**Comment by bk201** [comment]
The patch to managed chart works. We should also change the state from `modified` to `ready` here.
https://github.com/harvester/harvester/blob/75876deec220274d95041f1646ca83946d47d18f/package/upgrade/upgrade_manifests.sh#L365

**Comment by Vicente-Cheng** [comment]
> The patch to managed chart works. We should also change the state from `modified` to `ready` here.
> 
> https://github.com/harvester/harvester/blob/75876deec220274d95041f1646ca83946d47d18f/package/upgrade/upgrade_manifests.sh#L365

updated

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

File: package/upgrade/upgrade_manifests.sh [hunk 1]
```diff
@@ -362,7 +362,7 @@ EOF
   pause_managed_chart harvester-crd "false"
 
   # XXX The eventual state of harvester would be "modified", not "ready"!
-  wait_managed_chart fleet-local harvester $REPO_HARVESTER_CHART_VERSION $pre_generation_harvester modified
+  wait_managed_chart fleet-local harvester $REPO_HARVESTER_CHART_VERSION $pre_generation_harvester ready
```
Comment (bk201) [inline package/upgrade/upgrade_manifests.sh:364]: Please help remove the above comment. Thanks.

Comment (Vicente-Cheng) [reply package/upgrade/upgrade_manifests.sh:364]: done

**Comment by Vicente-Cheng** [comment]
hi @w13915984028, could you help to review this PR?  Thanks!

**Review by w13915984028 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**